NEWS.md

```diff
@@ -2,6 +2,11 @@
 
 ## Polars R Package (development version)
 
+### New features
+
+-   In the when-then-otherwise expressions, the last `$otherwise()` is now optional,
+    as in Python Polars (#836).
```

Comment:
```suggestion
-   In the when-then-otherwise expressions, the last `$otherwise()` is now optional,
    as in Python Polars. If `$otherwise()` is not specified, rows that don't respect the 
    condition set in `$when()` will be filled with `null` (#836).
```